### ISSUE TITLE
ADTPro 2.0.3 NOT opening on macOS Mojave

### ISSUE THREAD

**Issue by Woogetybop** [ADTPro/adtpro#62]
Unable to run ADTPro 2.0.3 java application with latest Java 8 on Mojave 14.2

Last login: Wed Jan 23 14:04:24 on ttys000
/Users/dad/Desktop/ADTPro-2.0.3/ADTPro.command ; exit;
OSx86-MacMini:~ dad$ /Users/dad/Desktop/ADTPro-2.0.3/ADTPro.command ; exit;
Exception in thread "main" java.lang.UnsupportedClassVersionError: org/adtpro/ADTPro : Unsupported major.minor version 52.0
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClassCond(ClassLoader.java:637)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:621)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:141)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:283)
	at java.net.URLClassLoader.access$000(URLClassLoader.java:58)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:197)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:190)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:306)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:301)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

VERSION 2.0.2 Opens correctly on Mojave 14.2

**Comment by david-schmidt** [comment]
I assume you mean 2.0.3?  Changing you title to reflect that.

I understand the `ADTPro.command` doesn't work for you.  2.0.3 is supposed to help alleviate the need for that by introducing changes in the app icon, so do please give that a whirl too.

At a terminal, what does `java -version` say to you?

**Comment by Woogetybop** [comment]
Yes my error. Version 2.0.3 not 2.0.2
Version 2.0.3 will not run by app or command.
I’m using version 2.0.2 right now just fine.

As for java version I checked at the terminal and that’s my problem. 32 bit java 6 JDK was active. I installed the JDK java 8 package and that fixed the issue. I had installed Java 8 JRE runtime, but 64bit JDK was required.

Thank you again for all your time and effort on what I consider a remarkable program. 

Best Regards,

Tod Sanders
woogetybop@gmail.com
